### ISSUE TITLE
When creating new collection hangs

### ISSUE THREAD

**Issue by Frozenfire92** [twitterdev/chrome-reporter-plugin#1]
Reproduce:
- Clone Repo
- Load unpacked extension
- Login
- Save pin
- Go to twitter, click + on tweet
- Create new collection
- Recieve following error

```
Settings.init complete                          page.js:12
extensions::uncaught_exception_handler:8 Error in event handler for (unknown): TypeError: Cannot read property 'timelines' of undefined
    at chrome-extension://lbdddbnfjjkkajkbndhjfokbhkcnhnkh/js/page.js:484:39handler @ extensions::uncaught_exception_handler:8
```

**Comment by ryankicks** [comment]
Hi Frozenfire92!

The Collections API is still under restricted access. If you submit a request here with your use case and details, we can get you going!

https://support.twitter.com/forms/platform

**Comment by ryankicks** [comment]
FWIW, a bug has been fixed so using the test tokens should now work. Would love to know how useful the feature is to you!

**Comment by Frozenfire92** [comment]
I wasn't all that interested in using the plugin. I just found out about it from [here](https://twittercommunity.com/t/chrome-extension-authentication-with-twitter/2494/6?u=frozenfire92). Making a chrome extension with twitter user oauth is painful to say the least due to lack of user oauth2. Upon finding this I have determined that this is by far the best solution there is, and would recommend you modify this and make it an official example.

Thanks for responding though, good to know the project wasn't dead :)
